### ISSUE TITLE
There's not a simple way to add a helm repo with a custom CA

### ISSUE THREAD

**Issue by ipedrazas** [fluxcd/helm-operator#17]
It's not easy to add your own repository configuration to the chart.

The setting `Values.helmOperator.configureRepositories.enable` allow you toinject a secret, however, if there's a repo that needs a CA there's not a standard way to add the CA into the deployment provided by the chart.

Note that `Values.helmOperator.tls.verify ` is used as a flag to configure `tiller` tls and not repo.

If you jump into the helmOperator pod and create the file `ca.crt` and then execute:

`helm repo add myrepo https://myhelmrepo.com --ca-file ca.crt`

the `HelmRelease` can use a helm repo with a self-signed cert. This means that the operator needs 2 things to install releases from a helm repo under a self-signed cert: 

* The CA certificate.
* The entry in the `repositories.yaml` with the location of the ca certificate.

**Comment by hiddeco** [comment]
I think this got resolved in #183 (cert path configuration is now possible, and the chart allows you to add additional mounts), and will be further improved with #142.

**Comment by kav** [comment]
Just wanted to leave a note in case anyone else goes down the rabbit hole here. It appears you should do something like the following.


First create a `repositories.yaml` as described in the docs. Add a line for that reads: 
```yaml
  ... 
  caFile: /root/.helm/certs/ca.crt
  ...
```
for any repository using your new ca.

Then run:
`kubectl create secret generic helm-repositories -n flux --from-file=repositories.yaml --from-file=ca.crt`

Then use these values with the helm-operator helm chart (presented as yaml but you can get crazy with command line opts if you want...
```yaml
...
configureRepositories:
  enable: true
  secretName: helm-repositories
extraVolumeMounts:
- name: repositories-yaml
  mountPath: /root/.helm/certs/ca.crt
  subPath: ca.crt
```
obviously you can tweak this for multiple certificate authorities or just move stuff around as you like.

Note: Your HelmRelease should continue to use the url of the repo _not the name_.
